variables.tf

```diff
@@ -0,0 +1,45 @@
+variable "resource_prefix" {
+  description = "Prefix to be used for naming new resources"
+  type        = string
+  default     = "cloudquery"
+}
+
+variable "aws_account_id" {
+  description = "AWS account ID"
```

Comment:
This is really "master" account ID. Please change it to reflect that.

Reply:
It is not only master account id, we also use this variable for  child accounts as well